core/src/main/resources/approval.properties

```diff
@@ -0,0 +1,61 @@
+############################################################################
+#                          APPROVAL.PROPERTIES                             #
+#    Configure the number of approvals required for each request Type      #
+#               Configure environment specific approvals                   #
+############################################################################
+#   Approval Types see io.aiven.klaw.model.enums.ApprovalTypes.java
+#   Options for approval settings
+#   RESOURCE_OWNER // This is the Topic owner for TOPIC,SCHEMA and ACl and the CONNECTOR owner for CONNECT
+#   ACL_OWNER //This will be ignored if added to anything other than an ACL/subscription
+#   TEAM_NAME // This is a place holder for a real team name. e.g. INFRATEAM
+
+
+#   Request Types see io.aiven.klaw.model.enums.RequestEntityType.java
+#   TOPIC
+#   SCHEMA
+#   ACL
+#   CONNECT
+
+#  Operation Types see io.aiven.klaw.model.enums.RequestOperationType.java
+#  CREATE
+#  UPDATE
+#  PROMOTE
+#  CLAIM
+#  DELETE
+#  ALL
+
+
+############################################################################
+#                          APPROVAL.PROPERTIES                             #
+#                           Default Settings                               #
+#                                                                          #
+############################################################################
+# TOPIC.ALL=RESOURCE_TEAM_OWNER
```

Comment:
```suggestion
# TOPIC.ALL=TOPIC_TEAM_OWNER
```

Reply:
I'll run through this here I thought I made the updates to the examples but I've obviously missed a few :(